src/main/java/pl/sda/javapoz/security/MyUserPrincipal.java

```diff
@@ -0,0 +1,51 @@
+package pl.sda.javapoz.security;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import pl.sda.javapoz.model.User;
+
+import java.util.Collection;
+
+public class MyUserPrincipal implements UserDetails {
+
+    private User user;
+
+    public MyUserPrincipal(User user) {
+        this.user = user;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
+    }
+
+    @Override
+    public String getPassword() {
+        return user.getPassword();
+    }
+
+    @Override
+    public String getUsername() {
+        return user.getEmail();
+    }
+
+    @Override
+    public boolean isAccountNonExpired() {
```

Comment:
Wiem ze to z poprzedniego projektu ale chyba nie jest potrzebne

Reply:
nie można tego wyrzucić bo w interfejsie to jest